### PR TITLE
Step by step navigation: adding documentation

### PR DESCRIPTION
**Pull request by Garneauma**
Adding .gc-stp-stp documentation and example.

### CODE REVIEW

File: components/gc-stp-stp/gc-stp-stp-en.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+		<ul class="toc lst-spcd col-md-12">
+			<li class="col-md-4 col-sm-6"><a class="list-group-item" href="index-en.html">1. [Step / section page name]</a></li>
+			<li class="col-md-4 col-sm-6"><a class="list-group-item active">2. [Step / section page name]</a></li>
+			<li class="col-md-4 col-sm-6 clr-lft-sm"><a class="list-group-item" href="page3-en.html">3. [Step / section page name]</a></li>
+			<li class="col-md-4 col-sm-6 clr-lft-md clr-lft-lg"><a class="list-group-item" href="page4-en.html">4. [Step / section page name]</a></li>
+			<li class="col-md-4 col-sm-6 clr-lft-sm"><a class="list-group-item" href="page5-en.html">5. [Step / section page name]</a></li>
+			<li class="col-md-4 col-sm-6"><a class="list-group-item" href="page6-en.html">6. [Step / section page name]</a></li>
+		</ul>
+	</div>
+</div>
```
Comment (duboisp) [inline components/gc-stp-stp/gc-stp-stp-en.html:30]: This don't like the version 2 of the component.

This is not a "toc" (table of content) if we compare on how the "toc" CSS is currently used for the in-page table of content component.

**Review by duboisp — CHANGES_REQUESTED** [review]
Would you like to create a include for this component and to update the "advanced service" template to use that include?

And I can see there is new CSS added for this component, is that necessary or probably a minor/patch change to the component?

**Comment by duboisp** [comment]
Pre-approved upon review and local testing.

To be confirm: The CSS code for the download link might need to be removed as it was already moved I think.

**Comment by Garneauma** [comment]
Removed the .gc-dwnld CSS. It was added by mistake during merge conflict resolution.

**Review by duboisp — APPROVED** [review]
Review and tested locally,

Work as expected. 

This is a minor change because it add another supported pattern for the gc step step which is the same as the version 1 but simpler to code for an author.

**Pull request merged**